br/pkg/backup/client.go

```diff
@@ -951,26 +954,29 @@ backupLoop:
 				zap.Int("retry time", retry))
 			return berrors.ErrFailedToConnect.Wrap(err).GenWithStack("failed to create backup stream to store %d", storeID)
 		}
-		defer bcli.CloseSend()
```

Comment:
can this be

```go
defer func() {
    _ = bcli.CloseSend()
}()
```

rather than copying the whole thing everything

Reply:
this defer is in a for loop. 
I will use another PR to refactor this function. https://github.com/pingcap/tidb/pull/31264